tests/contrib/botocore/test_bedrock_llmobs.py

```diff
@@ -282,6 +283,19 @@ def test_llmobs_converse(cls, bedrock_client, request_vcr, mock_tracer, llmobs_e
                 "output_tokens": response["usage"]["outputTokens"],
                 "total_tokens": response["usage"]["totalTokens"],
             },
+            tool_definitions=[
+                {
+                    "name": "fetch_concept",
+                    "description": "Fetch an expert explanation for a concept",
+                    "schema": {
+                        "json": {
+                            "type": "object",
+                            "properties": {"concept": {"type": "string", "description": "The concept to explain"}},
+                            "required": ["concept"],
+                        },
+                    },
+                }
+            ],
```

Comment:
Looks like this is the same tool definition for multiple tests. Can we make this a constant in `bedrock_utils.py` and reference this instead?